src/components/Mailbox.vue

```diff
@@ -102,6 +106,17 @@ export default {
 		envelopes() {
 			return this.$store.getters.getEnvelopes(this.account.id, this.folder.id, this.searchQuery)
 		},
+		hasMessages() {
```

Comment:
I know this is/was necessary in `MailboxMessage` but have you tried `return this.envelopes.length !== 0`? Since we use the list in other places inside this component, that should actually work just fine

Reply:
also because that is exactly what was used in the template :)